rules/attrs.bzl

```diff
@@ -34,6 +34,10 @@ ATTRS = dict(
         allow_single_file = True,
         doc = "Android manifest to run Android Lint against.",
     ),
+    android_home = attr.label(
```

Comment:
This should really live at the toolchain level instead of needing to be passed to every single target.